server/api/controllers/member.js/add-member.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
similar to the other comment i made in member.js model. Should we consider a more generic name instead of member? like user? also folder name should be member instead of member.js

Reply:
yeah will change to user actually